Sources/SignalRClient/ReconnectableConnection.swift

```diff
@@ -32,6 +32,7 @@ internal class ReconnectableConnection: Connection {
     var connectionId: String? {
         return underlyingConnection.connectionId
     }
+    var inherentKeepAlive: Bool = false
```

Comment:
This seems wrong. This would mean that again that we would start sending Pings for LongPolling. This should be a getter that forwards the setting from the `underlyingConnection`.